src/library/scala/collection/immutable/ListMap.scala

```diff
@@ -142,12 +159,23 @@ sealed class ListMap[A, +B] extends AbstractMap[A, B]
   protected class Node[B1 >: B](override protected val key: A,
                                 override protected val value: B1) extends ListMap[A, B1] with Serializable {
 
-    override def size: Int = sizeInternal(this, 0)
+    override def size: Int =
+      sizeInternal(this, 0)
 
     @tailrec private[this] def sizeInternal(cur: ListMap[A, B1], acc: Int): Int =
       if (cur.isEmpty) acc
       else sizeInternal(cur.next, acc + 1)
 
+    private[this] def sizeInternal1: Int = {
```

Comment:
Unused?

Reply:
I think it was used in https://github.com/scala/scala/pull/9215 - but this PR was split out to separate what can merge now